src/databricks/labs/blueprint/logger.py

```diff
@@ -66,11 +65,13 @@ def format(self, record: logging.LogRecord) -> str:
                 msg += "\n"
             msg += self.formatStack(record.stack_info)
 
-        color_marker = self.GRAY
-        if record.levelno in (logging.INFO, logging.WARNING):
-            color_marker = self.BOLD
-        elif record.levelno in (logging.ERROR, logging.FATAL):
-            color_marker = self.RED + self.BOLD
+        match record.levelno:
+            case logging.INFO | logging.WARNING:
```

Comment:
nit: Can we do WARNING a different color to INFO

Reply:
@sundarshankar89: I've refactored this a bit to make it (hopefully) clearer that there are two colors for each log-level.